pkgs/by-name/op/openfga-cli/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
```

```suggestion
{
  lib,
  buildGoModule,
  fetchFromGitHub,
  installShellFiles,
}:
```
Formatted as per RFC 0166. You can use the package `nixfmt-rfc-style` to automatically comply with the RFC in the future

---

Thanks, I missed that one, I was still using nixpkgs-fmt